src/app/static/src/tests/components/admin/settings.test.js

```diff
@@ -0,0 +1,65 @@
+import {shallowMount} from '@vue/test-utils';
+import {mockAxios} from "../../mockAxios";
+import Settings from "../../../js/components/admin/settings.vue";
+import ErrorInfo from "../../../js/components/errorInfo";
+
+describe("settings", () => {
+
+    const url =' http://app/settings/auth-allow-guest/';
+    beforeEach(() => {
+        mockAxios.reset();
+        mockAxios.onGet(url)
+            .reply(200, {"data": true});
+    });
+
+    it('gets allow guest setting and renders value', async () => {
+        const wrapper = shallowMount(Settings);
+        setTimeout(() => {
+            expect(mockAxios.history.get.length).toBe(1);
```

Comment:
It's so weird, code cov is complaining that this logic isn't covered, but is clearly is :woman_shrugging: